src/test/resources/test_scripts/freestylePing.sh

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-ping -c 10 spiegel.de
\ No newline at end of file
+ping -c 10 localhost
```

Comment:
Often the ping fails, and the test fails. could add `||true` at end but why not local host and don't need to hit the network.